charts/portainer/templates/NOTES.txt

```diff
@@ -1,20 +1,24 @@
-1. Get the application URL by running these commands:
+
 {{- if .Values.ingress.enabled }}
 {{- range $host := .Values.ingress.hosts }}
   {{- range .paths }}
-  http{{ if $.Values.ingress.tls }}s{{ end }}://{{ $host.host }}{{ if .port }}:{{ .port }}{{ else }}:9000{{ end }}{{.path}}
+  Use the URL below to access the application
+  http{{ if $.Values.ingress.tls }}s{{ end }}://{{ $host.host }}{{ if .port }}:{{ .port }}{{ else }}{{ end }}{{.path}}
```

Comment:
Have you tested how this works using a `helm install` locally? I.e, does it add anything weird to the URL by default (since 99% of the time, ingresses are listening on port 443 anyway)